consensus/vbft/msg_builder.go

```diff
@@ -183,7 +183,7 @@ func (self *Server) constructBlock(blkNum uint32, prevBlkHash common.Uint256, tx
 		txHash = append(txHash, t.Hash())
 	}
 	txRoot := common.ComputeMerkleRoot(txHash)
-	blockRoot := ledger.DefLedger.GetBlockRootWithNewTxRoot(txRoot)
+	blockRoot := ledger.DefLedger.GetBlockRootWithNewTxRoots(txRoot)
```

Comment:
txRoot of last-block should also be included here?

Reply:
yes, this is left for @xiemylogos to implement. so the title contains `[wip]`.

Reply:
ok, I am modifying